mailpoet/tasks/phpstan/custom-stubs.php

```
@@ -100,15 +100,44 @@ public function get_end() {
       public function get_customer_id() {
         return 1;
       }
+
+      /**
+       * Get meta data.
+       *
+       * @param string $key   Meta key.
+       * @param bool   $single Whether to return a single value.
+       * @return mixed
+       */
+      public function get_meta( $key, $single = true ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)